data-prepper-core/src/main/java/com/amazon/dataprepper/meter/EMFMetricUtils.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.amazon.dataprepper.meter;
+
+class EMFMetricUtils {
+
+    protected static final double MINIMUM_ALLOWED_VALUE = 8.515920e-109;
+    protected static final double MAXIMUM_ALLOWED_VALUE = 1.174271e+108;
+
+    static double clampMetricValue(final double value) {
```

Comment:
It might be good to add a Javadoc here that explains whatever restrictions are in place on the metrics values.